materialize-postgres/Dockerfile

```diff
@@ -10,8 +10,8 @@ COPY go.* ./
 RUN go mod download
 RUN go install github.com/mattn/go-sqlite3
 
-# Copy in a recent `flowctl` for usage by tests.
-COPY --from=ghcr.io/estuary/flow:dev /usr/local/bin/flowctl /usr/local/bin/flowctl
+# Download recent flow binaries for usage by tests.
+RUN curl -L --proto '=https' --tlsv1.2 -sSf "https://github.com/estuary/flow/releases/download/dev/flow-x86-linux.tar.gz"| tar -zx -C /usr/local/bin/ && cp /usr/local/bin/flowctl-go /usr/local/bin/flowctl
 
```

Comment:
Download the flow binaries, instead of fetch from image. And seems `flowctl-go` is what we need for now to get the tests pass.

Reply:
Might need some explanations here.
Right now if without any changes, the `/user/local/bin/flowctl` is the rust `flowctl`, which is not ready to be used for testing here. So the `cp` command here is to override the rust `/user/local/bin/flowctl` with go-based  `/user/local/bin/flowctl-go`, which is needed for testing now.  So this is not about copying files around, and I think later we will remove this command after rust `flowctl` is fully developed.

Reply:
I tested again with the downloaded rust `flowctl`, and it is working well. So the `cp` command is no longer needed here and everywhere.